lib/api/core.dart

```diff
@@ -192,10 +193,20 @@ class ApiConnection {
   }
 
   Future<T> postFileFromStream<T>(String routeName, T Function(Map<String, dynamic>) fromJson,
-      String path, Stream<List<int>> content, int length, {String? filename}) async {
+      String path, Stream<List<int>> content, int length, {String? filename,
+      String? contentType}) async {
```

Comment:
nit:
```suggestion
      String path, Stream<List<int>> content, int length,
      {String? filename, String? contentType}) async {
```

otherwise `path` and `contentType` look like the same kind of parameter, and it's a bit subtle to see that one is positional and the other named